packages/web-components/src/components/cbp-breadcrumb/cbp-breadcrumb.specs.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Breadcrumb/Specifications" />
+
+# cbp-breadcrumb
+
+## Purpose
+
+* Breadcrumb component exist to give users a key indicator of where they are within an application when the user happens to be deeper
```

Comment:
This is really too long for a purpose. It should be a sentence or two and not include guidance.